### ISSUE TITLE
False context wenn using wrapWith-Helper in wrapWith-Partial

### ISSUE THREAD

**Issue by chaenu** [Sebastian-Fitzner/mangony-hbs-helper-wrap-with#1]
Hey,

I'm using a wrapWith helper in a template:

```
{{#wrapWith "test"}}
    my test content
{{/wrapWith}}
```

and the wrapWith partial looks like this:

```
<div class="test">
    {{{yield}}}
</div>
```

Now I want to use another wrapWith helper inside of this partial:

```
<div class="test">
    {{#wrapWith "new-test"}}
        my new test content
    {{/wrapWith}}

    {{{yield}}}
</div>
```

but the problem is, that the yield context is overwritten with the content defined inside of the new wrapWith Helper "my new test content". Expected output is "my test content". If I change the source order and include the new wrapWith Helper after the yield, everything works as expected:

```
<div class="test">
    {{{yield}}}

    {{#wrapWith "new-test"}}
        my Content
    {{/wrapWith}}
</div>
```

Do you have any idea to fix this?

**Comment by Sebastian-Fitzner** [comment]
The problem is the context merging. `{{{yield}}}` and `{{options}}` gets merged with the root context. 

Fixed it in v0.0.2.

**Comment by chaenu** [comment]
Yes, everything works as expected. Thank you!
